app/src/main/java/edu/uc/groupProject/topten/service/FirestoreService.kt

```diff
@@ -297,6 +301,9 @@ class FirestoreService {
         return result
 
 
+    }
 
+    fun setIncrementTime(countdownTime: Long){
```

Comment:
I  added this function as I think it is preferred that FirestoreService is also encapsulated from the ViewModels. Thus, I prefer that ViewModel calls this function instead of making the assignment itself.
(of course, this is the 2nd step, the first step was to not have the fragment do this assignment, but rather call a function MainViewModel).